test/indexer/test_anvil.py

```diff
@@ -65,15 +65,18 @@ def bundle_fqid(cls,
                                   entity_type=entity_type)
 
     @classmethod
-    def bundles(cls) -> list[TDRAnvilBundleFQID]:
-        return [
-            cls.bundle_fqid(uuid='826dea02-e274-affe-aabc-eb3db63ad068',
-                            version='')
-        ]
+    def primary_bundle(cls) -> TDRAnvilBundleFQID:
+        return cls.bundle_fqid(uuid='826dea02-e274-affe-aabc-eb3db63ad068')
 
-    @property
-    def bundle(self) -> TDRAnvilBundleFQID:
-        return one(self.bundles())
+    @classmethod
+    def supplementary_bundle(cls) -> TDRAnvilBundleFQID:
+        return cls.bundle_fqid(uuid='6b0f6c0f-5d80-a242-accb-840921351cd5',
+                               entity_type=BundleEntityType.supplementary)
+
+    @classmethod
+    def dataset_bundle(cls) -> TDRAnvilBundleFQID:
```

Comment:
```suggestion
    def duos_bundle(cls) -> TDRAnvilBundleFQID:
```

and elsewhere. I requested this nomenclature in a review to an earlier PR of yours.

Reply:
> and elsewhere

I cannot find any other occurrences.